packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -158,3 +163,62 @@ class RabbitAutoscalingStatusMessage(_RabbitAutoscalingBaseMessage):
     instances_running: int = Field(
         ..., description="the number of EC2 instances currently in running state in AWS"
     )
+
+
+class _RabbitResourceTrackingBaseMessage(RabbitMessageBase):
+    channel_name: Literal["io.simcore.simcore.tracking"] = Field(
+        default="io.simcore.simcore.tracking", const=True
+    )
+
+    service_run_id: str = Field(
+        ..., description="uniquely identitifies the service run"
+    )
+    created_at: datetime.datetime = Field(..., description="message creation datetime")
+
+    def routing_key(self) -> str | None:
+        return None
+
+
+class RabbitResourceTrackingStartedMessage(_RabbitResourceTrackingBaseMessage):
+    wallet_id: PositiveInt
```

Comment:
There is a WalletID
`from models_library.wallets import WalletID`

Reply:
good.. was not available when I created the PR ;)

Reply:
aa you are right :) i need to merge my PR, so then tomorrow it will be :)

Reply:
ok I'll wait till tomorrow.